cmd/vic-machine/create/create.go

```diff
@@ -1273,10 +1274,29 @@ func (c *Create) generateCertificates(server bool, client bool) ([]byte, *certif
 	return cakp.CertPEM, skp, nil
 }
 
-func logArguments(cliContext *cli.Context) {
+func (c *Create) logArguments(cliContext *cli.Context) []string {
+	args := []string{}
+
 	for _, f := range cliContext.FlagNames() {
 		if !cliContext.IsSet(f) {
-			continue
+			skip := true
+			// This is a workaround for cli.Context.IsSet() returning false when
+			// the short option for a cli.StringSlice is supplied instead of the long option
+			// see https://github.com/urfave/cli/issues/314
+			create := reflect.ValueOf(c).Elem()
+			for i := 0; i < create.NumField(); i++ {
+				t := create.Type().Field(i)
+				if flag := t.Tag.Get("flag"); flag == f {
+					ss := create.Field(i)
+					if !ss.IsNil() {
+						skip = false
+						break
+					}
+				}
+			}
+			if skip {
+				continue
+			}
 		}
 
 		// avoid logging sensitive data
```

Comment:
user password is not persisted here.
probably should think about encrypting the whole command in guestinfo, so keep sensitive data is not an issue.